src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer is working.");
+    }
+
+    @Override
```

Comment:
Please remove the redundant empty line after the method signature (line 9). According to the checklist, there should not be an empty line after the method signature.